package.json

```diff
@@ -42,6 +44,7 @@
     "typescript": "^3.2.2"
   },
   "dependencies": {
-    "node-pre-gyp": "^0.12.0"
+    "node-pre-gyp": "^0.12.0",
+    "yargs": "^13.2.4"
```

Comment:
Since this is just used in a test script, I think it can safely be moved to `devDependencies`.